litgpt/generate/base.py

```diff
@@ -78,7 +78,8 @@ def generate(
     top_k: Optional[int] = None,
     top_p: float = 1.0,
     eos_id: Optional[int] = None,
-) -> torch.Tensor:
+    stream: bool = False,
```

Comment:
the above StreamLitAPI class seem to be missing `generate(..., stream=True)` argument.